gammapy/data/pointing.py

```diff
@@ -88,6 +90,55 @@ def _str_for_index(self, idx):
         ss += 'ALTAZ: {} deg\n'.format(self.altaz[idx].to_string())
         return ss
 
+    def _str_for_time(self, t):
+        """Information for an arbitrary time"""
+
+        if t < self.time[0]:
+            return 'Pointing at time {} was requested but history begins at {}'.format(t.mjd, self.time[0].mjd)
```

Comment:
Instead of returning a string with the error message, you should raise an error:
```
raise ValueError('Pointing at time {} was requested but history begins at {}'.format(t.mjd, self.time[0].mjd))
```